Patch to add Solaris 12 to the list of supported OSes.  This patch is a
remnant of one that has been submitted upstream.  Part of the submission 
seems to have made it upstream but not this remaining part.

--- puppetlabs-ntp-4.1.2/metadata.json.orig     2016-04-01 08:28:52.379882695 -0700
+++ puppetlabs-ntp-4.1.2/metadata.json  2016-04-01 08:30:33.122129237 -0700
@@ -101,5 +101,5 @@
       "version_requirement": ">= 3.0.0 < 5.0.0"
     }
   ],
-  "description": "NTP Module for Debian, Ubuntu, CentOS, RHEL, OEL, Fedora, FreeBSD, ArchLinux and Gentoo."
+  "description": "NTP Module for Debian, Ubuntu, CentOS, RHEL, OEL, Fedora, FreeBSD, ArchLinux, Gentoo, and Solaris."
 }
